app/frontend/src/features/auth/useAuthStyles.ts

```diff
@@ -108,31 +110,29 @@ const useAuthStyles = makeStyles((theme) => ({
   },
   formWrapper: {
     [theme.breakpoints.up("md")]: {
-      backgroundColor: "#f8f7f4",
-      borderRadius: "4px",
+      backgroundColor: "#f8f7f4", // TODO: check with the palette
+      borderRadius: `${theme.shape.borderRadius / 3}px`,
```

Comment:
```suggestion
      borderRadius: theme.shape.borderRadius / 3,
```
Pretty sure this works and the px gets added for you

Reply:
Yeah it does, I've seen in code somewhere that we add px sometimes hence I added it, removing it then.

Reply:
There are certain properties it doesn't (or didn't) work for, I think it was a jss bug